pkg/customparams/customparams.go

```diff
@@ -121,36 +121,45 @@ func (p *CustomParams) GetParams(ctx context.Context) (map[string]string, map[st
 				"ParamsFilterUsedValue",
 				fmt.Sprintf("repo %s, param name %s has a value and secretref, picking value", p.repo.GetName(), value.Name))
 		}
-		if value.Value != "" {
-			ret[value.Name] = value.Value
-		} else if value.SecretRef != nil {
+
+		_, paramIsStd := stdParams[value.Name]
```

Comment:
should we check that the key in stdParams and parsedFromComment actually exist in there? (i am surprise golangci-lint didn't bug you about this)

Reply:
That's what we're doing here, actually. At this point in the code we don't care about the value, just if the param exists in the stdParams and parsedFromCOmment maps, so we're checking the `ok` return-value instead of the map's value at that key.